functests/1_performance/netqueues.go

```diff
@@ -53,6 +54,9 @@ var _ = Describe("[ref_id: 40307][pao]Resizing Network Queues", func() {
 	})
 
 	BeforeEach(func() {
+		if discovery.Enabled() && testutils.ProfileNotFound {
+			Skip("Discovery mode enabled, performance profile not found")
+		}
 		profile, err := profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
 		Expect(err).ToNot(HaveOccurred())
 		if profile.Spec.Net == nil {
```

Comment:
in general, I am unsure we should run such tests under the `1_performance` test suite, like in case when we will update a customer performance profile and for some reason, the rollback will fail, we can get a lot of troubles.

I am proposing to skip the test if the profile does not have some specific net fields and set fields under the performance profile used for CI.

It probably should not be part of this PR, but @mrniranjan can you please open the following PR?

Reply:
@cynepco3hahue  i can move the netqueues to functests/2_performance_update. if that's okay ? In this automation we tests the network fields . So it can't be  set in performance profile before hand.

Reply:
it will work for me, but still we need to have something under 1_performance test

Reply:
@cynepco3hahue can you approve this PR, for splitting the netqueues, i will open a new PR .